webapp-client/src/App.js

```diff
@@ -36,10 +37,7 @@ export default class App extends React.Component {
   render() {
     return (
       <Fragment>
-        <div className="App">
-          <h1>Lobster Ice Cream</h1>
-          <img src={logo} className="App-logo" alt="logo" />
-        </div>
+        <Navbar logo={logo} />
```

Comment:
Is there a reason why we are passing in the logo here rather than directly linking it in Navbar.js? just wondering in terms design choice

Reply:
I think I may have overengineered that. I was thinking that if we use it anywhere else on the page, this would be useful. I don't know where else on the page we'd use it tho 😂